### DISCUSSION TITLE
support for ILI9485 needed..

### DISCUSSION THREAD

**Discussion by yo2ldk** [Bodmer/TFT_eSPI#2336] [New processor/display support or capability request]
Hi Bodmer,
 I use with success your excellent library, with different TFT display's
  Now I have a new one, and I need to use it in my radio project, it help me because is a bit larger, 4.3"
   so, please if you can add drivers on library for this or is one equivalent ( I try it some but not work ) : 
   HMI 8M PSRAM 16M Smart Display Screen ,
   ILI9485 driver, it have on it ESP32 S3,
   It have driver just on Arduino GFX and thi si for him:
   
   ` #include <Arduino_GFX_Library.h>

#define LCD_PWR_PIN 2          
or
#define TFT_BL 2

#define GFX_BL DF_GFX_BL 
#if defined(DISPLAY_DEV_KIT)
Arduino_GFX *gfx = create_default_Arduino_GFX();
#else 
Arduino_ESP32RGBPanel *bus = new Arduino_ESP32RGBPanel(
    GFX_NOT_DEFINED /* CS */, GFX_NOT_DEFINED /* SCK */, GFX_NOT_DEFINED /* SDA */,
    40 /* DE */, 41 /* VSYNC */, 39 /* HSYNC */, 42 /* PCLK */,
    45 /* R0 */, 48 /* R1 */, 47 /* R2 */, 21 /* R3 */, 14 /* R4 */,
    5 /* G0 */, 6 /* G1 */, 7 /* G2 */, 15 /* G3 */, 16 /* G4 */, 4 /* G5 */,
    8 /* B0 */, 3 /* B1 */, 46 /* B2 */, 9 /* B3 */, 1 /* B4 */
);

 Arduino_RPi_DPI_RGBPanel *gfx = new Arduino_RPi_DPI_RGBPanel(
   bus,
   800 /* width */, 0 /* hsync_polarity */, 8 /* hsync_front_porch */, 4 /* hsync_pulse_width */, 8 /* hsync_back_porch */,
   480 /* height */, 0 /* vsync_polarity */, 8 /* vsync_front_porch */, 4 /* vsync_pulse_width */, 8 /* vsync_back_porch */,
   1 /* pclk_active_neg */, 16000000 /* prefer_speed */, true /* auto_flush */); 
#endif /* !defined(DISPLAY_DEV_KIT) */
/*******************************************************************************
 * End of Arduino_GFX setting
 ******************************************************************************/ 
 
[ESP32-8048S043 Specifications-EN.pdf](https://github.com/Bodmer/TFT_eSPI/files/10451259/ESP32-8048S043.Specifications-EN.pdf)

 Thank you in advance !!

**Comment by Bodmer** [comment]
That display has an RGB interface which requires a full frame buffer to be continuously sent to the display. That interface type is not supported by the library and support is unlikely to be added in the near future.
The library does support the SSD1963 display with an 8 bit interface (16 bit on RP2040 only), displays with that controller are available in larger screen sizes and pixel resolutions.

**Reply by yo2ldk** [reply]
I see  that is RGB..
 Thank you for reply  that it is !

**Comment by andreaschizzali** [comment]
I believe that the driver ILI9485 does not exist.
I have a similar display model ESP32-487S043 with the same problem.
In the documentation from Sunton about the display module there are interesting documents.
In the specifications of the screen (4.3” 480X272-TN bare screen specs.pdf), the company Shenzen Jincai Intelligent Co., LTD states: "Driver IC  ILI6485". In the document ESP32-4827S043 Specifications-EN.pdf the same company states "Driver IC  ILI9485" in the document ESP32-4827S043 Specifications-EN.pdf.
It seems to me there is merely a simple transcription error, so the effective driver chip is the ILI6485.
The datasheet for the ILI6485 can be easily found (https://aitendo3.sakura.ne.jp/aitendo_data/product_img/lcd/tft2/QT43272B576/ILI6485_DTS_V001_20180403.pdf). The informations from the datasheet correspond to the values employed in the demo programs.
I hope that helps.
